chromium_src/chrome/browser/prefs/browser_prefs.cc

```diff
@@ -109,6 +112,11 @@ void MigrateObsoleteProfilePrefs(Profile* profile) {
 #if BUILDFLAG(ENABLE_CUSTOM_BACKGROUND)
   NTPBackgroundPrefs(profile->GetPrefs()).MigrateOldPref();
 #endif
+
+  auto* map = HostContentSettingsMapFactory::GetForProfile(profile);
```

Comment:
I'd like to test that this actually runs, but pulling this into the the unit test would be a layering violation.